linux/deploy_vm/deploy_vm_from_iso.yml

```diff
@@ -187,6 +187,9 @@
       when:
         - unattend_install_conf is defined
         - unattend_install_conf is match('Photon') or unattend_install_conf is match('Ubuntu')
+    
+    - name: "Get VM's primary network adapter MAC address after poweron VM"
+      include_tasks: ../../common/vm_wait_primary_nic_mac.yml
```

Comment:
Missing `when: vm_primary_nic_mac is undefined or not vm_primary_nic_mac`

Reply:
Done. Thanks